### DISCUSSION TITLE
Julia: How to use Plots.jl with Plotly backend to show interactive plots in Revealjs

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#2218] [Q&A]
I'm trying to use create a [Revealjs presentation](https://quarto.org/docs/presentations/revealjs) and the code snippets that produce plots are using [Julia](https://julialang.org/) with the [Plots.jl](https://docs.juliaplots.org/stable/) package and the [Plotly backend](https://docs.juliaplots.org/stable/backends). The problem I'm running into is that upon rendering with quarto, the code plotting snippets save/inject the generated plots as static files. There was [an example with a note](https://quarto.org/docs/interactive/widgets/jupyter.html#plotly) using Plotly in the context of python/jupyter notebooks, but not clear how this translates to Julia code snippets using Plots.jl within VSCode. Does anyone have previous troubleshooting on this? In addition, for R it looks like one can use the Shiny package to create dashboard slides, maybe something similar can be done using [Genie.jl](https://genieframework.com/#genie-section)/[Stipple.jl](https://genieframework.com/#stipple-section). Guess the question is how can we make dashboard slides in Revealjs with Julia code?

Here is an example of the code block in a `.qmd` file:

````julia
```{julia}
#| echo: true
#| label: fig-sinewave
#| fig-cap: "Sine wave"
#| column: margin
using Plots; plotly()  # or plotlyjs()
x = LinRange(0.0,2π,100)
plot(x,sin.(x)) 
```
````

then when I render this with quarto in Revealjs format the slide plot is a stored as a SVG file.

**Comment by RoyiAvital** [comment]
Did you find a solution to use `plotly` with Julia in Quarto?
